worlds/smw/Regions.py

```diff
@@ -1152,10 +2157,10 @@ def add_location_to_region(world: MultiWorld, player: int, active_locations, reg
 
 
 
-def connect(world: MultiWorld, player: int, used_names: typing.Dict[str, int], source: str, target: str,
+def connect(multiworld: MultiWorld, player: int, used_names: typing.Dict[str, int], source: str, target: str,
```

Comment:
can't comment on it specifically since it's old code, but the entire rest of this method can pretty much be replaced with just `source_region.connect(target_region, target_region.name, rule)`